pkg/apis/management.cattle.io/v3/pod_security_types.go

```diff
@@ -0,0 +1,92 @@
+package v3
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const PodSecurityAdmissionConfigurationTemplateKind = "PodSecurityAdmissionConfigurationTemplate"
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type PodSecurityAdmissionConfigurationTemplate struct {
```

Comment:
Since `PodSecurityPolicyTemplate` is in `authz_types.go`, would this make sense to have these types there as well? 
This is really just a question, if there is a rationale for it being in own file, that's fine,

Reply:
Jack and I agreed to put it into a new file, it might be helpful if more helper functions are needed for PSS / PSA.